integration_tests/__tests__/jest.config.js-test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const {extractSummary, cleanup, writeFiles} = require('../utils');
+const path = require('path');
+const runJest = require('../runJest');
+
+const DIR = path.resolve(__dirname, '../jest.config.js');
```

Comment:
Looks like this directory name is confusing for Travis? Maybe we can go with `js-config` or fix it otherwise?